cmd/lakectl/cmd/log.go

```diff
@@ -1,7 +1,9 @@
 package cmd
```

Comment:
I think that the example in the PR description should be documented in the `Long` field of the cli command.

Reply:
Will add!